### PR TITLE
add job history to judge application status #2625

### PR DESCRIPTION
**Pull request by itbasketplayer**
## What is the purpose of the pull request

add job history status url to judge application status, when application number threshold is reached(default 10000,maybe it was set to 1000).
you can see issue #2625

### CODE REVIEW

**Comment by itbasketplayer** [comment]
it is my problem?

File: dolphinscheduler-common/src/main/resources/common.properties [hunk 1]
```diff
@@ -18,47 +18,49 @@
...
 
-# if resource.storage.type=S3，s3 access key
-fs.s3a.access.key=A3DXS30FO22544RE
+# if resource.storage.type=S3,s3 access key
+#fs.s3a.access.key=A3DXS30FO22544RE
 
-# if resource.storage.type=S3，s3 secret key
-fs.s3a.secret.key=OloCLq3n+8+sdPHUhJ21XrSxTC+JK
+# if resource.storage.type=S3,s3 secret key
+#fs.s3a.secret.key=OloCLq3n+8+sdPHUhJ21XrSxTC+JK
```
Comment (dailidong) [inline dolphinscheduler-common/src/main/resources/common.properties:55]: why comment these lines?

**Comment by dailidong** [comment]
> it is my problem?

you are right

**Comment by dailidong** [comment]
> > it is my problem?
> 
> you are right

sorry，accidentally closed the question， reopen again

File: dolphinscheduler-common/src/main/resources/common.properties (hunk 1 above)
Comment (itbasketplayer) [reply dolphinscheduler-common/src/main/resources/common.properties:55]: it is idea format code :)

**Comment by itbasketplayer** [comment]
@dailidong should reopen again? :)

**Comment by dailidong** [comment]
sorry for review so late，could you submit this PR content to dev branch ?

**Comment by qiaozhanwei** [comment]
submit this PR  to dev branch. Thx

**Comment by dailidong** [comment]
sorry for not merge into dev-1.3.1

could you submit this PR to dev branch ? 

sorry again

**Comment by itbasketplayer** [comment]
ok,i will do it on weekends :)

**Comment by yangyichao-mango** [comment]
Hi,
Please resolve the conflict. Thx a lot~

**Comment by lgcareer** [comment]
@itbasketplayer Hi,thanks for you contribution,you can resend the pr to the dev branch,And if you resent,you can paste the pr url here,then we will review the pr on dev and close this.

**Comment by CalvinKirs** [comment]
Hi, please resolve the conflict.. If there is no update after 7 days, the PR will be closed, and you can reopen it when your problem is resolved.

**Comment by chengshiwen** [comment]
duplicated with [#2848]

**Comment by dailidong** [comment]
> duplicated with [#2848], so thir pr should be closed.

I will close this PR

**Pull request closed without merging**